transport/nio/pom.xml

```diff
@@ -61,6 +61,59 @@
             <artifactId>jmock-legacy</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
This dependency is actually better going off under the k3po-maven-plugin section.  WS didn't do this because it had k3po methods being pulled in the specification that were used by k3po tests written in transport ws.